src/storage/v2/indices/vector_index.cpp

```diff
@@ -0,0 +1,196 @@
+// Copyright 2024 Memgraph Ltd.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+// License, and you may not use this file except in compliance with the Business Source License.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+#include <cstdint>
+#include <thread>
+
+#include "absl/container/flat_hash_map.h"
+#include "storage/v2/id_types.hpp"
+#include "storage/v2/indices/vector_index.hpp"
+#include "storage/v2/vertex.hpp"
+#include "usearch/index.hpp"
+#include "usearch/index_dense.hpp"
+#include "utils/algorithm.hpp"
+
+// NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
+DEFINE_HIDDEN_string(experimental_vector_indexes, "",
+                     "Enables vector search indexes on nodes with Label and property specified in the "
+                     "IndexName__Label1__property1__{JSON_config};IndexName__Label2__property2 format.");
```

Comment:
I think the second option currently doesn't work -> make it work or update (there is another related comment)

![Screenshot 2024-10-23 at 8 44 07 AM](https://github.com/user-attachments/assets/c2a21778-c6c8-40bb-af34-731d2f4badbd)